src/UserRepository.js

```diff
@@ -4,7 +4,12 @@ class UserRepository {
     this.avgUserStepGoal = null;
   }
   findUserData(id) {
-    return this.allUserData[id - 1];
+    const userData = this.allUserData.find((data) => {
+      if (data.id === id) {
```

Comment:
Remove if statement, can just have data => data.id === id

Reply:
just saw this but I have already refactored